AcloConcurrency/AcloConcurrency/Collection/Cocktails/CocktailDescription/Views/CocktailDescriptionView.swift

```diff
@@ -0,0 +1,123 @@
+import SwiftUI
+
+struct CocktailDescriptionView: View {
+
+    let image: String
+    let name: String
+
```

Comment:
use private let and Init

Reply:
It's just for mock. We will need a modelView in future

Reply:
Remove it please